### PR TITLE
Switch from cordova-universal-links-plugin to cordova-universal-links…

### PR DESCRIPTION
**Pull request by nicolasgarnier**
…-plugin-fix

See https://github.com/firebase/firebaseui-web/issues/219

### CODE REVIEW

File: packages/auth/src/cordovahandler.js [hunk 1]
```diff
@@ -171,12 +171,12 @@ fireauth.CordovaHandler.prototype.initializeAndWait = function() {
...
   this.isReady_ = fireauth.util.checkIfCordova().then(function() {
     // Check all dependencies installed.
-    // https://github.com/nordnet/cordova-universal-links-plugin
+    // https://www.npmjs.com/package/cordova-universal-links-plugin-fix
     var subscribe = fireauth.util.getObjectRef(
         'universalLinks.subscribe', goog.global);
     if (typeof subscribe !== 'function') {
       throw fireauth.CordovaHandler.getError_(
-          'cordova-universal-links-plugin is not installed');
+          'cordova-universal-links-plugin-fix is not installed');
```
[the remark this answers is not in the document]
Comment (nicolasgarnier) [reply packages/auth/src/cordovahandler.js]: both done

File: packages/auth/src/cordovahandler.js [hunk 2]
```diff
@@ -171,12 +171,12 @@ fireauth.CordovaHandler.prototype.initializeAndWait = function() {
   }
   this.isReady_ = fireauth.util.checkIfCordova().then(function() {
     // Check all dependencies installed.
-    // https://github.com/nordnet/cordova-universal-links-plugin
+    // https://www.npmjs.com/package/cordova-universal-links-plugin-fix
```
[the remark this answers is not in the document]
Comment (nicolasgarnier) [reply packages/auth/src/cordovahandler.js]: Done

**Comment by nicolasgarnier** [comment]
@bojeil-google done.

PTAL

**Comment by bojeil-google** [comment]
Thanks again for the fix!

**Pull request merged**